Source/SantaGUI/SNTMessageWindowController.h

```diff
@@ -23,13 +24,18 @@
 ///
 ///  Controller for a single message window.
 ///
-@interface SNTMessageWindowController : NSWindowController
+@interface SNTMessageWindowController : NSWindowController<NSApplicationDelegate,
+                                                           WebPolicyDelegate,
+                                                           WebFrameLoadDelegate>
+@property(weak) IBOutlet NSWindow *mainWindow;
+@property(weak) IBOutlet WebView *webView;
+
 
 - (instancetype)initWithEvent:(SNTStoredEvent *)event andMessage:(NSString *)message;
 
 - (IBAction)showWindow:(id)sender;
 - (IBAction)closeWindow:(id)sender;
-- (IBAction)showCertInfo:(id)sender;
+- (IBAction)showCertInfo;
```

Comment:
Remove these 3 methods - the first is defined in NSWindowController, the second is being changed (see my comment in the implementation) and the last is only used within the class.